README.md

```diff
@@ -8,7 +8,7 @@ See [K8S.md](K8S.md)
 
 ## Testing kubevirt with locally provisioned cluster
 
-See [KUBEVIRT_LOCAL_TESTING.md](KUBEVIRT_LOCAL_TESTING.md)
+See [KUBEVIRTCI_LOCAL_TESTING.md](KUBEVIRTCI_LOCAL_TESTING.md)
```

Comment:
maybe worth to change also line 9

Reply:
[fixed](https://github.com/kubevirt/kubevirtci/compare/2799113dc1c1b4c0c7a9babb44129833e0a16745..6a8e6e930340757e6ca4003f924f5aef6161a9ce)